Pod/Classes/SuperMock.swift

```diff
@@ -28,7 +28,23 @@ public class SuperMock: NSObject {
         NSURLSession.sharedSession().configuration.protocolClasses?.append(SuperMockURLProtocol)
         
         SuperMockResponseHelper.bundleForMocks = bundle
+        
+    }
+    
+    class func beginRecording(bundle: NSBundle?, mocksFile: String? = "Mocks.plist", policy: SuperMockResponseHelper.RecordPolicy) {
+        
+        NSURLProtocol.registerClass(SuperMockRecordingURLProtocol)
+        NSURLSessionConfiguration.defaultSessionConfiguration().protocolClasses = [SuperMockRecordingURLProtocol.self]
+        NSURLSession.sharedSession().configuration.protocolClasses?.append(SuperMockRecordingURLProtocol)
+        
+        SuperMockResponseHelper.bundleForMocks = bundle
+        SuperMockResponseHelper.sharedHelper.recording = true
+        
+    }
```

Comment:
I think beginRecording should be public, so it could be used in obj-c. (same for endRecording)